probe-rs/src/bin/probe-rs/cmd/dap_server/server/debugger.rs

```diff
@@ -835,6 +836,7 @@ impl Debugger {
             supports_instruction_breakpoints: Some(true),
             supports_stepping_granularity: Some(true),
             supports_completions_request: Some(true),
+            // support_terminate_debuggee: Some(true),
```

Comment:
I'm curious, why did you decide to disable this? The current `disconnect` request handling in the adapter checks the optional argument that is enabled by this.

Reply:
I think I confused it with the `supportsTerminateRequest`, I'll enable it again.